pages/about/contributing-in-github.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Contributing in GitHub
```

Comment:
I'd suggest to use via instead of in.

Reply:
changed _in_ -> _via_ throughout, even file name